core/domain/skill_domain.py

```diff
@@ -1148,13 +1298,20 @@ def _convert_rubric_v1_dict_to_v2_dict(cls, rubric_dict):
         Returns:
             dict. The converted rubric_dict.
         """
-        explanation = rubric_dict['explanation']
-        del rubric_dict['explanation']
+        # Using type ignore[misc] below because the Rubric domain object
+        # contains an explanations field instead of explanation
+        # Hence RubricDict has an explanations key instead of explanation
+        # but the below code is using explanation.
+        explanation = rubric_dict['explanation'] # type: ignore[misc]
+        del rubric_dict['explanation'] # type: ignore[misc]
```

Comment:
Ah, interesting, so the previous schema of rubric contained `explanation` but now it contains `explanations`, not sure if this is the right solution, but I will have to think about it.

Reply:
Yep. Is there any alternative fix for this or should I leave this as it is for now?

Reply:
@vojtechjelinek Could you please guide me on how to resolve this issue?

Reply:
Lets just keep the ignores.

Reply:
Done